### DISCUSSION TITLE
Savvycan did run and now won't in win10

### DISCUSSION THREAD

**Discussion by TurboStew** [collin80/SavvyCAN#738] [General]
Savvycan was working, now it wont run.  It spins for a sec when clicking on exe then nothing.  reboot, diff version of savvvycan, ran as admin, etc, etc.  will not run.... any ideas?

**Comment by collin80** [comment]
Can you run it from the command line and copy/paste what it says? It might give some sort of crash info. If not then we have less to go on but that's still some information.

**Reply by ghost** [reply]
I am having the same problem, but when I run it from the command line it opens fine (no errors or any more info), so that's an okay work around. Thanks

**Reply by VK2HFG** [reply]
Perhaps there's no path for the installed directory, which was the reason why my installation didn't initially run.
